### DISCUSSION TITLE
Is it possible to change the aud scope sent by Zitadel ?

### DISCUSSION THREAD

**Discussion by perrze** [zitadel/zitadel#7147] [Q&A]
Hi everyone !
I'm using ZITADEL as a SSO for my org and it works very well.
Only problem is that I plan using https://github.com/warp-tech/warpgate as a SSH bastion and this little open source has a misconception.
To check that an OIDC issuer is the good one it reads the aud scope in the JWT and awaits that it only contains the OIDC client-id.
But Zitadel is not only using the client-id but also the project ID and that blocks this tool to use it.

In this context I want to know if it's possible to **change how ZITADEL sends aud scope with Actions or such things ?**

Thanks for your help !

**Comment by hifabienne** [comment]
@peintnermax @stebenz can you help here?

**Comment by t3cneo** [comment]
Same problem with proxmox...

I might be wrong but to my understanding, as of now, the 'aud' part of a token can't be stripped down, [discussed here](https://github.com/zitadel/zitadel/discussions/6926#discussioncomment-7978070) in an other discussion about actions

**Comment by perrze** [comment]
I up this in case anyone has an idea

**Reply by fforootd** [reply]
Somebody just recently asked if we would accept a PR to disable this. 

But so far we have not had a PR or I have not seen it 😅
